data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -148,72 +155,99 @@ public AbstractVerticle get(final DataPlaneContract.Resource resource, final Dat
 
     final BaseConsumerVerticle.Initializer initializer = (vertx, consumerVerticle) ->
       authProvider.getCredentials(resource).onSuccess(credentials -> {
-        KafkaClientsAuth.attachCredentials(consumerConfigs, credentials);
-        KafkaClientsAuth.attachCredentials(producerConfigs, credentials);
-
-        final KafkaConsumer<Object, CloudEvent> consumer = createConsumer(vertx, consumerConfigs);
-        AutoCloseable metricsCloser = Metrics.register(consumer.unwrap());
-
-        final var egressConfig =
-          egress.hasEgressConfig() ?
-            egress.getEgressConfig() :
-            resource.getEgressConfig();
-
-        final var egressSubscriberSender = createConsumerRecordSender(
-          vertx,
-          egress.getDestination(),
-          egressConfig
-        );
-
-        final var egressDeadLetterSender = hasDeadLetterSink(egressConfig)
-          ? createConsumerRecordSender(vertx, egressConfig.getDeadLetter(), egressConfig)
-          : NO_DEAD_LETTER_SINK_SENDER;
-
-        final var filter = egress.hasFilter() ?
-          new AttributesFilter(egress.getFilter().getAttributesMap()) :
-          Filter.noop();
-
-        final var responseHandler = getResponseHandler(egress,
-          () -> getResponseToKafkaTopicHandler(vertx, producerConfigs, resource),
-          () -> new ResponseToHttpEndpointHandler(createConsumerRecordSender(vertx, egress.getReplyUrl(), egressConfig)));
-        final var commitIntervalMs = Integer.parseInt(String.valueOf(consumerConfigs.get(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG)));
-
-        final var recordDispatcher = new RecordDispatcherMutatorChain(
-          new RecordDispatcherImpl(
-            new ResourceContext(resource, egress),
-            filter,
-            egressSubscriberSender,
-            egressDeadLetterSender,
-            responseHandler,
-            new OffsetManager(vertx, consumer, eventsSentCounter::increment, commitIntervalMs),
-            ConsumerTracer.create(
-              ((VertxInternal) vertx).tracer(),
-              new KafkaClientOptions()
-                .setConfig(consumerConfigs)
-                // Make sure the policy is propagate for the manually instantiated consumer tracer
-                .setTracingPolicy(TracingPolicy.PROPAGATE)
+          KafkaClientsAuth.attachCredentials(consumerConfigs, credentials);
+          KafkaClientsAuth.attachCredentials(producerConfigs, credentials);
+
+          final KafkaConsumer<Object, CloudEvent> consumer = createConsumer(vertx, consumerConfigs);
+          AutoCloseable metricsCloser = Metrics.register(consumer.unwrap());
```

Comment:
Mind checking your IDE settings for the project to use the `.editorconfig` file in the repo?

Reply:
Done. I had to manually revert though, that change is not related to the .editorconfig, it's some setting in Intellij